.gitattributes

```diff
@@ -0,0 +1 @@
+* text=auto eol=lf
```

Comment:
what does this do?

Reply:
It let’s your IDE know you have to use LF line endings